var/spack/repos/builtin/packages/mlperf-opencatalyst/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class MlperfOpencatalyst(PythonPackage, CudaPackage):
```

Comment:
Personally I would name this `py-ocp-models` since that's the name used in https://github.com/mlcommons/hpc/blob/main/open_catalyst/setup.py

Reply:
Done